ansible_commands/shc_ready.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/python
+'''
+Ansible module
+
+This module will be called to wait until SHC is completely setup and 
+initial bundle replication has occurred such that user defined 
+bundles are safe to push
+'''
+import time
+import json 
+import requests
+
+from ansible.module_utils.basic import AnsibleModule
+
+class ShcReady(object):
+    def __init__(self, module):
+        self.captain_url = module.params["captain_url"]
+        self.shc_peers = module.params["shc_peers"]
+        self.user = module.params["spl_user"]
+        self.password = module.params["spl_pass"]
+        self.retry_times = module.params["retry_times"]
+
+    def run(self):
+        URL = "https://{0}:8089/services/shcluster/status?output_mode=json".format(self.captain_url)
+        peers_online = False
+        peers_ready = False 
+        time.sleep(60) # give the SHC some initial time to setup
```

Comment:
Do we want to make these sleeps configurable?  Might be something worth tweaking

Reply:
I don't think I'd start with a sleep, but rather attempt the connection first and if it fails go in to a sleep loop. There are going to be cases where the SHC is already up and this is an unnecessary delay.